### ISSUE TITLE
[Glitch] Issue with displaying some unicode characters

### ISSUE THREAD

**Issue by ashtonx** [Zren/plasma-applet-eventcalendar#37]
Had it after recent update, can anyone check if it can be reproduced?
my display is:

`📆 dd/MM/yy 🕗 hh:mm`

I'm missing the calendar unicode. Seems to be only issue with the widget as everything else displays it properly.. Then again not excluding potential plasma issue.

my locale.conf

```
GNU nano 2.9.8                                                                                 /etc/locale.conf

LANG=en_US.UTF-8
LC_CTYPE=en_US.UTF-8
LC_NUMERIC=pl_PL.UTF-8
LC_TIME=pl_PL.UTF-8
LC_COLLATE=pl_PL.UTF-8
LC_MONETARY=pl_PL.UTF-8
LC_MESSAGES="en_US.UTF-8"
LC_PAPER=pl_PL.UTF-8
LC_NAME=pl_PL.UTF-8
LC_ADDRESS=pl_PL.UTF-8
LC_TELEPHONE=pl_PL.UTF-8
LC_MEASUREMENT=pl_PL.UTF-8
LC_IDENTIFICATION=pl_PL.UTF-8
LC_ALL=
```

**Comment by Zren** [comment]
Works for me in Plasma 5.13.5 in KDE Neon when testing in `plasmoidviewer` using the the polish locale. I tested in `plasmashell` too, but I'm not going to fuss with my system locale yet.

![](https://i.imgur.com/qfnJMgo.png)

![](https://i.imgur.com/nBKRXno.png)

1. What's you're desktop font? Noto Sans?
2. Do you have color emoji's?

**Comment by ashtonx** [comment]
Yeah, noto sans, used to work with others too. No color though.
Dedicated emoji font works though...

After seeing this I tried some other plasma apps I didn't use and they also seem to have the issue. (ironically terminal, browser and some other software works). I'm guessing something went wrong with update or removing orphans..

Seems like it's an issue with my display manager config and I just happened to update at the same time as event calendar. 

Sorry for bothering :)
